cmd/boost/direct_deal.go

```diff
@@ -30,16 +31,14 @@ var directDealAllocate = &cli.Command{
 	Usage: "Create new allocation[s] for verified deals",
```

Comment:
Details: "", // neeeds the csv format listed here.

Reply:
Added description with details about CSV format